docs/_data/repositories/development.yml

```diff
@@ -0,0 +1,25 @@
+section:
+  description: Many of our internal development tools are also open source. Take
+    a look - we're happy to share, in case any might also solve a problem you have.
+  categories:
+  - name: Development Tools
+    description:
+    repos:
+      - name: cyberark/bash-lib
+        url: https://github.com/cyberark/bash-lib
+        description: Bash library for functions and scripts that are used in
+          pipelines for multiple repos.
+      - name: cyberark/helm-charts
+        url: https://github.com/cyberark/helm-charts
```

Comment:
I don't feel like this fits in this tab.  This is for installing `cyberark/conjur-oss-helm-chart`?  Seems like it should go under "Conjur Core".

Reply:
it's a collection of helm charts for all of our tools, though it only holds conjur oss for now it could hold helm charts for secretless and potentially other future tooling.